lib/components/fullpage.js

```diff
@@ -40,12 +40,12 @@ class Fullpage extends React.Component {
   }
 
   componentDidMount() {
-    document.addEventListener('wheel', this.onScroll.bind(this));
-    document.addEventListener('touchstart', this.onTouchStart.bind(this));
-    document.addEventListener('touchend', this.onTouchEnd.bind(this));
-    document.addEventListener('keydown', this.checkKey.bind(this));
-    window.addEventListener('resize', this.onResize.bind(this));    
-    events.pub(this, this.scrollToSlide);    
+    document.addEventListener('wheel', this.onScroll);
```

Comment:
Ah i see, yeah an oversight by me. Glad you found this by switching between apps. If you wouldn't mind could you just rewrite the componentDidUnmount lifecycle to use binds instead of adding to webpack configs for stage 2? I'm sure they'll likely be accepted, i'd just rather keep it as vanilla as possible for now.